metabase/metabase
# Hide read-only items in the Archive

**Problem:** Read-only items - items belonging to read-only collections - are appearing in users' archives. This causes frustration because they are shown items they can perform no actions against. Additionally, showing the items (as is) provide no context/value to the user.

**Context:**
- https://github.com/metabase/metabase/issues/24018

---
**Desired Behavior**/**Acceptance Criteria**

_Behaviors_
- [ ] Only actionable items - curate-able items - are shown in the archive
  - [ ] Read-only items are hidden
- [ ] Archive has no other altered behaviors

_Tests_
- [ ] Test added to verify curate-able items are **shown** and read-only items are **not shown** to users'
- [ ] Test added to verify admins can see all items

---
**How to Verify**

1. Log in as admin 
3. Create read-only collection > add question to read-only collection > archive the question
4. Create curate-able collection > add question to curate-able collection > archive the question
5. View the archive > see that both items appear and you can perform actions against both
  ![image](https://github.com/metabase/metabase/assets/22608765/4032c0c6-b04c-47c7-b82f-f2e7d8ea49d2)
6. Setup the archive again with a read-only item and curate-able item
7. Log in as read-only user
8. View the archive > see that only the curate-able item appears and that you are able to unarchive it
  ![image](https://github.com/metabase/metabase/assets/22608765/87364257-ca38-4970-b8c2-ee483f5b7eba)
